### ISSUE TITLE
python-memcached 1.53 wheel package has 1.55 code in it

### ISSUE THREAD

**Issue by vkurup** [linsomniac/python-memcached#72]
Hi,

I just noticed that some of my TravisCI tests started failing. They depend on `python-memcached==1.5.3`, and the failure looks like this:
```
ERROR: test_cache_is_working (bulk_sms.tests.test_sending.MultiprocSendingTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/<projectname>/bulk_sms/tests/test_sending.py", line 330, in test_cache_is_working
    self.assertTrue(cache.get('CACHETEST', default=False),
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/django/core/cache/backends/memcached.py", line 82, in get
    val = self._cache.get(key)
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/memcache.py", line 1089, in get
    return self._get('get', key)
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/memcache.py", line 1073, in _get
    return _unsafe_get()
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/memcache.py", line 1061, in _unsafe_get
    value = self._recv_value(server, flags, rlen)
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/memcache.py", line 1235, in _recv_value
    val = int(buf)
ValueError: invalid literal for int() with base 10: 'True'
```

When I look at [this page](https://pypi.python.org/pypi/python-memcached/1.53), the source package has different code than the wheel package:

![image](https://cloud.githubusercontent.com/assets/4413/8910832/0377d236-3457-11e5-80e8-4f47d0501579.png)

Specifically, line 93 of memcache.py inside the wheel package says this:

```
__version__ = "1.55"
```

Would it be possible to upload a new wheel package with 1.53 code in it? Thanks for your work modernizing this package!

**Comment by kmaglione** [comment]
This is currently breaking our tests as well. The other builds on PyPI all seem to contain the correct version.

**Comment by kevinastone** [comment]
:+1:

**Comment by t0m** [comment]
The zip file is unaffected so you can install the correct version with:

pip install python-memcached==1.53 --no-index --find-links https://pypi.python.org/packages/source/p/python-memcached/python-memcached-1.53.tar.gz#md5=89570d26e7e7b15caa668a6b2678bd3c

**Comment by dieselmachine** [comment]
The setup.py files for 1.54, 1.55, and 1.56 contain

    version="1.53",

I think this is what is causing pypi to return the most recent version even when 1.53 is explicitly requested. I think editing the setup.py to reflect the correct version number would solve the issues people are having with testing.

**Comment by linsomniac** [comment]
Fixed?  I accidentally pushed a newer version as a wheel with the 1.53 tag.  :-(  I deleted that wheel from pypi.

**Comment by dieselmachine** [comment]
The setup.py files still contain an older version number

Example: 1.57 branch setup.py

https://github.com/linsomniac/python-memcached/blob/release-1.57/setup.py

Note the 'version' value there is 1.53, that should probably match the __version__ value in memcache.py

**Comment by linsomniac** [comment]
I've modified my release process so that it commits the version into the repo rather than changing it as part of the post-tagging build and upload.  So for 1.58 it should be fixed, but it seems a shame to release a 1.58 just to roll that.  I'm open to the idea of doing it though.
